src/modules/sensors/vehicle_angular_velocity/imu_gyro_parameters.c

```diff
@@ -120,4 +120,4 @@ PARAM_DEFINE_INT32(IMU_GYRO_RATEMAX, 0);
 * @reboot_required true
 * @group Sensors
 */
-PARAM_DEFINE_FLOAT(IMU_DGYRO_CUTOFF, 0.0f);
```

Comment:
Can you keep this in a separate commit?

Reply:
Sure, although I mainly made the change to prompt the discussion.